egs/apiai_decode/s5/download-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
+
+DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
+
```

Comment:
If we will go into versioning, then https://api.ai/downloads/api.ai-kaldi-asr-model.zip will be latest version (link), and other versions will have numbers at the end.

Also we will track everything here https://github.com/api-ai/api-ai-english-asr-model